src/components/HelpResourceCard.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+
+import {
+  Card,
+  CardHeader,
+  CardBody,
+  CardFooter,
+  Link,
```

Comment:
instead of Link from react-uswds import Link from react-router-dom. They have the same name, so it might be helpful to do something like `import {Link as RouterLink} from react-router-dom` and then use the `RouterLink` instead of `Link`